### PR TITLE
Void Region Fixes

### PR DESCRIPTION
**Pull request by pshriwise**
The recent update for `NOT_FOUND` --> `-2` has not propagated here. This PR takes care of that. 

This now allows void regions to appear as something other than background in the plotter, but also requires us to handle a couple special cases. This PR addresses that by always adding a void `DomainView` to the materials `Domain` with id `_VOID_REGION`. Void regions can now be masked, highlighted, and colored like any other region and is accessible via all the standard menus in the GUI.

### CODE REVIEW

File: plotmodel.py [hunk 1]
```diff
@@ -308,6 +310,14 @@ def getDomains(self, file, type_):
             domain = DomainView(id, name, color, masked, highlighted)
             domains[id] = domain
 
+        # always add void to a material domain at the end
+        if 'material' in file:
+            void_id = str(_VOID_REGION)
+            domains[void_id] = DomainView(void_id, "VOID",
+                                          (255, 255, 255),
```
Comment (paulromano) [inline plotmodel.py]: Do we want to be able to have the color be configurable?

**Comment by pshriwise** [comment]
Added a quick note on usage in the README here too.

File: plotmodel.py (hunk 1 above)
Comment (pshriwise) [reply plotmodel.py]: Hrmmm I think it's as configurable as any other color in the plotter. It just doesn't get a random color assigned to start with - always defaults to white. Am I missing something?

Comment (paulromano) [reply plotmodel.py]: Nope, you're right. Thanks for pointing that out!

**Pull request merged**